src/data/polynomial.lean

```diff
@@ -609,6 +539,89 @@ begin
     finset.sum_mul_boole, finset_sum_coeff, ite_le_nat_degree_coeff],
 end
 
+section map
```

Comment:
Could you explain why this moved? Probably something changed but it's not easy to identify at a glance.

Reply:
It's because I added `mem_map_range` to the bottom of the section. It needed later stuff, but I thought it would be nice to keep these `map`-related things together. So I moved the whole `map`-section down a bit.